giles-eco/src/main/java/edu/asu/diging/gilesecosystem/web/core/files/IProcessingRequestsDatabaseClient.java

```diff
@@ -14,5 +14,12 @@ public interface IProcessingRequestsDatabaseClient extends IDatabaseClient<IProc
     public abstract List<IProcessingRequest> getProcRequestsByRequestId(String procReqId);
 
     public abstract List<IProcessingRequest> getIncompleteRequests();
+    
+    /**
 
-}
\ No newline at end of file
+    Deletes processing requests associated with the specified document ID.
+    @param documentId The ID of the document for which processing requests should be deleted.
+    */
+    public abstract void deleteProcessingRequestsForDocumentId(String documentId);
```

Comment:
`deleteByDocumentId` or `deleteRequestsByDocumentId` is sufficient; it's the processing request database client, so it's expected that it's processing requests.